### PR TITLE
TS-1308

### PR DESCRIPTION
**Pull request by hmcginnis**
:pencil: indicate truncation behavior

### CODE REVIEW

File: content/user/apps/Session-metadata.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Pinpoint the source of vulnerabilities in your application with session metadata
 
 ## Agent Configuration 
 
-To send session metadata for your application to the Contrast UI, you must add the configuration settings to your agent configuration file. The build properties that the agent can report are **branch name**, **build number**, **commit hash**, **committer**, **Git tag**, **repository**, **test run** and **version**. You may include all or some of these properties, as desired. See the following table for the configuration key that corresponds to each property. 
+To send session metadata for your application to the Contrast UI, you must add the configuration settings to your agent configuration file. The build properties that the agent can report are **branch name**, **build number**, **commit hash**, **committer**, **Git tag**, **repository**, **test run** and **version**. You may include all or some of these properties, as desired. The value of each property, up to a maximum of 255 characters, will be stored as filterable, identifiable information associated with each vulnerability reported during the session. See the following table for the configuration key that corresponds to each property.
```
Comment (evamill) [inline content/user/apps/Session-metadata.md]: Could be more clear. Like:
The system can store up to 255 characters for each build property. This metadata will be available to you as additional information for each vulnerability reported or a way to filter them.

**Comment by gdvdsn-contrast** [comment]
@hmcginnis @kristianamitchellcontrastsecurity Should the Session Metadata page already be published on the docs site (under Contrast UI > Applications)? I don't see it in the left nav. 
Screen Shot 2019-12-17 at 2 18 16 PM

**Comment by hmcginnis** [comment]
@gdvdsn-contrast , it is a published document. it's a section under `View Vulnerabilities` - https://docs.contrastsecurity.com/user-vulnerableapps.html#session

**Comment by hmcginnis** [comment]
@gdvdsn-contrast , just to close the loop on this, now that there's UX approval, do I merge this or will your team?

**Comment by gdvdsn-contrast** [comment]
@hmcginnis Thanks for your help. I will merge and get it published. For now, it's easier for the writers to keep track of what's being merged, so we can close the publishing loop. We may change that in the future, but for now I think it's best to let us know when it's the right time to publish.

**Comment by gdvdsn-contrast** [comment]
This page has been updated and published: https://github.com/Contrast-Security-OSS/docs/blob/master/content/user/apps/Session-metadata.md.

**Pull request merged**